### PR TITLE
Incubator: teacher homepage banner & page

### PR DESCRIPTION
**Pull request by breville**
This adds a new banner to the teacher homepage ("en" only) at https://studio.code.org/home, and a new page at https://studio.code.org/incubator.

If necessary, the banner on the teacher homepage can be hidden with the `"hide_incubator_link"` DCDO flag.

It also adds the ability for a `HeaderBanner` to have an image on the right side, on wider displays, used here for the lightbulb image.  It flips to the left for RTL.

### Teacher homepage

Screenshot 2022-10-25 at 4 22 48 PM

### Incubator page

Screenshot 2022-10-25 at 4 11 58 PM

### CODE REVIEW

File: apps/src/sites/studio/pages/incubator/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore} from '@cdo/apps/code-studio/redux';
+import Incubator from '@cdo/apps/templates/studioHomepages/Incubator';
+
+$(document).ready(function() {
+  ReactDOM.render(
+    <Provider store={getStore()}>
```
Comment (molly-moen) [inline apps/src/sites/studio/pages/incubator/index.js:10]: are we using redux at all in the `Incubator` component?

**Review by molly-moen — APPROVED** [review]
LGTM!

File: apps/src/sites/studio/pages/incubator/index.js (hunk 1 above)
Comment (breville) [reply apps/src/sites/studio/pages/incubator/index.js:10]: Great question.  The `HeaderBanner` component apparently [needs](https://github.com/code-dot-org/code-dot-org/blob/f8c00060368b0c9f9cf0132e0a7cfce8f58949b9/apps/src/templates/HeaderBanner.jsx) it.

**Review by sanchitmalhotra126 — APPROVED** [review]
Looks good!

**Review by sanchitmalhotra126 — APPROVED** [review]

**Review by molly-moen — APPROVED** [review]

**Pull request merged**